wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/model/impl/UserImpl.java

```diff
@@ -8,52 +8,64 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.api.user.server.dao;
+package org.eclipse.che.api.user.server.model.impl;
```

Comment:
it's always good even if usually we see quickly when we fails in these methods

but for example with copy/paste we could return another field for a given method or set the wrong field